src/battle_util.c

```diff
@@ -8347,7 +8345,7 @@ u8 IsMonDisobedient(void)
     if (BattlerHasAi(gBattlerAttacker))
         return 0;
 
-    if (IsBattlerModernFatefulEncounter(gBattlerAttacker)) // only false if illegal Mew or Deoxys
+    if (TRUE) // This used to be a check to see if the mon in question was a fateful encounter (Mew or Deoxys)
     {
         if (gBattleTypeFlags & BATTLE_TYPE_INGAME_PARTNER && GetBattlerPosition(gBattlerAttacker) == B_POSITION_PLAYER_RIGHT)
```

Comment:
Why not check everything in the new function? The check in this function seems redundant. I would move the full check to before the battler does anything so it isn't scattered around.

Reply:
Now that you mention it, yea.
I think I started with this approach when I thought I only had 1 bit to use in `SpecialStatus`